### PR TITLE
README.md: Markdown standardization

### PR DESCRIPTION
**Pull request by pesekon2**
* delete trailing spaces
* set line length to 80 characters
* delete multiple spaces after hashes in titles
* delete trailing punctuation in titles
* avoid bare URLs
* use code block style for code blocks
* delete them peculiar spaces appearing at the beginning of some lines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,88 @@
...
+```
 
 ## How to install or remove AddOns in your GRASS installation
 
-The simplest way to install GRASS GIS AddOns is to use the `g.extension` 
-module: \
-https://grass.osgeo.org/grass78/manuals/g.extension.html
+The simplest way to install GRASS GIS AddOns is to use the `g.extension`
+module:
+<https://grass.osgeo.org/grass78/manuals/g.extension.html>
```
Comment (neteler) [inline README.md:27]:
```suggestion
<https://grass.osgeo.org/grass-stable/manuals/g.extension.html>
```

File: README.md [hunk 2]
```diff
@@ -1,76 +1,88 @@
...
 
-  Preparations (assuming source code in $HOME/grass78/):
-    (if you have already build GRASS from source you don't need to do this
-    again. If adding to a binary install the versions must match exactly.
-    For a git clone this means that the main GRASS binary and source
-    code versions (GRASS GIS 6 or 7) must match.)
-    
-    ./configure # [opionally flags]
-    make libs
+Preparations (assuming source code in $HOME/grass78/):
```
Comment (neteler) [inline README.md:33]:
```suggestion
Preparations (assuming the source code in $HOME/grass78/):
```

File: README.md [hunk 3]
```diff
@@ -1,76 +1,88 @@
...
+(if you have already build GRASS from source you don't need to do this
```
Comment (neteler) [inline README.md:34]:
```suggestion
(if you have already built GRASS from source you don't need to do this
```

Comment (pesekon2) [reply README.md:34]: I have also noticed that and wanted to - for the sake of the git history purity - open another PR for that, but yes, I guess it is just an annoying overhead to split these small changes into fifteen smaller ones.

**Pull request merged**